openedx/core/djangoapps/call_stack_manager/tests/models.py

```diff
@@ -0,0 +1,42 @@
+"""
+The file defines Model classes required for testing call_stack_manager.py
+"""
+from django.db import models
+from openedx.core.djangoapps.call_stack_manager import CallStackManager, CallStackMixin
+
+
+class ModelMixinCSM(CallStackMixin, models.Model):
+    """
+    Test Model class which uses both CallStackManager, and CallStackMixin
+    """
+    # override Manager objects
+    objects = CallStackManager()
+
+    id_field = models.IntegerField()
+    string_field = models.TextField()
+
+
+class ModelMixin(CallStackMixin, models.Model):
+    """
+    Test Model that uses CallStackMixin but does not use CallStackManager
+    """
+    id_field = models.IntegerField()
+    text_field = models.CharField(max_length=10)
+    float_field = models.FloatField()
+
+
+class ModelNothing(models.Model):
+    """
+    Test Model class that neither uses CallStackMixin nor CallStackManager
+    """
+    id_field = models.IntegerField()
+    name_field = models.CharField(max_length=20)
+
+
+class ModelAnotherCSM(models.Model):
+    """
+    Test Model class that only uses overridden Manager CallStackManager
+    """
+    objects = CallStackManager()
+    id_field = models.IntegerField()
+    name_field = models.CharField(max_length=20)
```

Comment:
Since the `CallStackManager` and `CallStackMixin` don't actually do anything with the fields of the model, you could just not assign any fields to the models at all.